### ISSUE TITLE
Train inception

### ISSUE THREAD

**Issue by kushia** [fchollet/deep-learning-models#19]
Hi everyone,
I'm trying to train the inception model from custom images.
But I got one problems..
Here is the code I use 
`import os
import numpy as np
import pandas as pd
from scipy.misc import imread
from scipy.misc import imresize
from inception_v3 import InceptionV3
import keras

root_dir = os.path.abspath('.')
data_dir = os.path.join(root_dir, 'data')
os.path.exists(root_dir)
os.path.exists(data_dir)

train = pd.read_csv(os.path.join(data_dir, 'images', 'test.csv'))
train.head()
temp = []
for img_name in train.filename:
    image_path = os.path.join(data_dir, 'images', img_name)
    img_tmp = imread(image_path)
    img = imresize(img_tmp, (299, 299))
    img = img.astype('float32')
    temp.append(img)

train_x = np.stack(temp)
split_size = int(train_x.shape[0]*0.7)

train_x, val_x = train_x[:split_size], train_x[split_size:]
train_y, val_y = train.label.ix[:split_size],train.label.ix[split_size:]
inception_model = InceptionV3(weights=None)
inception_model.compile(loss='categorical_crossentropy', optimizer='adam', metrics=['accuracy'])
trained_model = inception_model.fit(train_x, train_y, nb_epoch=1, batch_size=8)
trained_model.save('car_model.h5')
print ('Fin de l\'entrainement')
`

But I got this error : 
Exception: Error when checking model target: expected predictions to have shape (None, 1000) but got array with shape (7, 1)

Do you know why ?
I think I did all good..

**Comment by prithi** [comment]
You need to exclude the logistic layer of inception as ImageNet has 1000 class and add your own logistic layer. 

Here is a example - https://github.com/fchollet/keras/blob/master/docs/templates/applications.md#fine-tune-inceptionv3-on-a-new-set-of-classes

**Comment by kushia** [comment]
Hi !
Thanks, this made the job, but I got an other problems, still with inception model..
I open a new issues, we can close this one :-)
